docs/getting_started/install.md

```diff
@@ -17,6 +17,21 @@ make
 ```
 For example:  `-DLP_SOLVE=/usr/lib/lpsolve/liblpsolve55.so`
 
+To install few dependencies before `make` command, run the following,
```

Comment:
Could we be more specific here? `ieeefp.h` is needed for `qd` library, right? An this is only needed in your setting which is some ubuntu box oven windows, right? If this is the case please separate the statements below and write one part for `qd` e.g. "the following could be needed in some architectures" and then for mkl "the following could be useful for building with `mkl` support"

Reply:
Updated. Please have a look!